tasks.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""
+Task execution tool & library
+"""
+
+import re
+import sys
+from logging import basicConfig, getLogger
+from pathlib import Path
+
+import json
```

Comment:
we require git python?  is that always there?

Reply:
ok, it is in the piffle, still we should document

Reply:
It's all in virtual env; only need python3 and pipenv, and only as a maintainer performing release actions